src/open_r1/utils/model_utils.py

```diff
@@ -0,0 +1,26 @@
+from transformers import AutoTokenizer, PreTrainedTokenizer
+
+from trl import ModelConfig
+
+from ..configs import GRPOConfig, SFTConfig
+
+
+DEFAULT_CHAT_TEMPLATE = "{% for message in messages %}\n{% if message['role'] == 'user' %}\n{{ '<|user|>\n' + message['content'] + eos_token }}\n{% elif message['role'] == 'system' %}\n{{ '<|system|>\n' + message['content'] + eos_token }}\n{% elif message['role'] == 'assistant' %}\n{{ '<|assistant|>\n'  + message['content'] + eos_token }}\n{% endif %}\n{% if loop.last and add_generation_prompt %}\n{{ '<|assistant|>' }}\n{% endif %}\n{% endfor %}"
```

Comment:
Use ChatML by default. Not sure if I should prefill the `<think>` token since that's a bit dataset dependent